site/_resources/how-we-work.md

```diff
@@ -25,16 +33,52 @@ Everything flows from this statement.
 - [Talk about what you intend to do, then do the thing you talked about.][1]
 GitHub review tools suck for extended debate, if you find you’re taking past your reviewer, its a sign that more design is needed.
 
-- Log an issue or it didn’t happen. 
```

Comment:
Yeh I agree with this. With this labels revamp, I'm trying to bring more structure and visibility to issue triage so that when people file issues the issues don't get lost.

Anyway, this text moved under the "Github Issue Management" heading.